lib/archiving.js

```diff
@@ -287,26 +305,32 @@ exports.startArchive = function (ot, config, sessionId, options, callback) {
     function startArchiveCallback(err, response, body) {
       if (err) {
         callback(err);
+        return;
       }
-      else if (response.statusCode !== 200) {
-        if (response && response.statusCode === 404) {
+      try {
+        body = JSON.parse(body);
+      } catch {}
```

Comment:
The linter complains that this (and other instances of `catch {}`) results in a parsing error. (Replace with `catch (error) { }`). 

There are already existing linting errors in the main branch. But if you add `/* eslint-env es2018 */` to the top of this file, you will see more linting errors.